torch/nn/modules/transformer.py

```diff
@@ -84,9 +84,9 @@ def forward(self, src: Tensor, tgt: Tensor, src_mask: Optional[Tensor] = None, t
         Shape:
             - src: :math:`(S, N, E)`.
             - tgt: :math:`(T, N, E)`.
-            - src_mask: :math:`(S, S)`.
-            - tgt_mask: :math:`(T, T)`.
-            - memory_mask: :math:`(T, S)`.
+            - src_mask: :math:`(S, S)` or :math:`(N*nhead, S, S)`.
```

Comment:
Better to be consistent between `nhead` and `num_heads`.

Reply:
Definitely. I went with what was already there. The Transformer documentation uses `nhead` https://pytorch.org/docs/master/generated/torch.nn.Transformer.html

The MultiHeadAttention docs use `num_heads`. https://pytorch.org/docs/master/generated/torch.nn.MultiheadAttention.html.

I did not diverge from these.

Reply:
For MHA, we should use `num_heads`.

Reply:
Yes I use `num_heads` in MHA. You'll notice that I use `nhead` in transformer.py and `num_heads` in activation.py